### PR TITLE
Get all tweets of user

### PR DESCRIPTION
**Pull request by Noisyboy-9**
In this pr, we are trying to get the list of all tweets for a specific user. The route for doing so is /users/student_id/tweets.

### CODE REVIEW

File: src/db/queries/tarin/update.js [hunk 1]
```diff
@@ -1,24 +1,24 @@
-const User = require('../../models/user');
+const user = require('../../models/user');
```
Comment (arefehkompani) [inline src/db/queries/tarin/update.js:1]: based on your previous comments I thought it is conventional to start model names with small letter, is it so?
I will revert this change, if it is conventional to use small letters just tell me and I will make new changes.

File: src/controllers/user.controller.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+    getTweetsOfUser(req, res) {
+        //check if user is logged in
+        new authMiddleware(req, res).getUser();
+
+        const studentId = req.params.id;
+
+        userService.getTweetsByStudentId(studentId)
+            .then(tweets => {
+                logger.info(`Get list of all tweets by student id ${studentId} successfully`);
```
Comment (arefehkompani) [inline src/controllers/user.controller.js]: `logger` is not defined

File: src/db/queries/tweets/read.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+const Tweet = require('../../models/tweets');
+module.exports = {
+    async getTweetsByUserId(id) {
```
Comment (arefehkompani) [inline src/db/queries/tweets/read.js]: We don't need `async` here.

File: src/db/queries/user/read.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+const User = require('../../models/user');
+
+module.exports = {
+    getAll() {
+        return User.find({}, ['name', 'studentId', 'avatar']);
+    },
+
+    async getUserByStudentId(studentId) {
```
Comment (arefehkompani) [inline src/db/queries/user/read.js]: We don't need `async` here too.

**Comment by Noisyboy-9** [comment]
changes requested have been implemented, please take a look and give me another review

**Review by arefehkompani — APPROVED** [review]

**Pull request merged**